code/modules/clothing/spacesuits/rig/suits/mining.dm

```diff
@@ -0,0 +1,36 @@
+/obj/item/weapon/rig/mining
```

Comment:
Add this rig to the loot list in random/rig

also add it to the contents of /obj/structure/closet/secure_closet/planet_cracker
found in code/game/objects/structures/crates_lockers/closets/secure/mining.dm